t/01-sanity.t

```diff
@@ -18,8 +18,22 @@ our $HttpConfig = qq{
     lua_package_path "$pwd/lib/?.lua;;";
 };
 
+our $HttpConfigWithInit = qq{
+    lua_package_path "$pwd/lib/?.lua;;";
+
+    init_by_lua_block {
+        local l = require("resty.lmdb")
+
+        local res, err = l.set("test", "value")
+
+        package.loaded.res = res
+        package.loaded.err = err
+    }
+};
+
 no_long_string();
 #no_diff();
+no_shuffle();
```

Comment:
what's the reason of adding this? (what will happen if the test is done in random order as it currently does?)

Reply:
fix